app/scripts/components/root-about/index.js

```diff
@@ -1,63 +1,34 @@
+import { PageMainContent } from '$styles/page';
+import PageHero from '$components/common/page-hero';
+import { FoldProse } from '$components/common/fold';
+import MdxContent from '$components/common/mdx-content';
 
-import PageHero from '../common/page-hero';
-import { FoldProse } from '../common/fold';
+const aboutContent = config.pageOverrides?.aboutContent;
 
 function RootAbout() {
   return (
     <PageMainContent>
       <LayoutProps title='Thematic about' />
       <PageHero
-        title='About the Dashboard'
-        description='Visualization, Exploration, and Data Analysis (VEDA): Scalable and Interactive System for Science Data.'
+        title={aboutContent?.data.title || 'About the Dashboard'}
+        description={
+          aboutContent?.data.description || 'Lorem ipsum dolor sit amet.'
+        }
       />
-      <FoldProse>
-        <p>
-          The VEDA Dashboard is one of several user interfaces in the VEDA
-          project. It also is aligned with VEDA’s goal of advancing interactive
-          visualization and exploratory visual data analysis. This dashboard
-          constitutes:
-        </p>
-        <ul>
-          <li>
-            an evolution of COVID-19 dashboard to provide the interactive
-            storytelling for various environmental changes using Earth
-            observation data;
-          </li>
-          <li>
-            a transformation of high-valued NASA datasets to dynamic
-            visualization, enabling users to perform on-the-fly visual analysis;
-          </li>
-          <li>
-            support for environmental justice initiatives through the
-            integration of NASA Earth observation data and socio-economic data;
-          </li>
-          <li>
-            a tool by which users can visually explore and localize data, and
-            perform independent analysis; data-driven stories that are
-            exportable in GIS formats;
-          </li>
-          <li>
-            and a situational awareness system that brings together Earth
-            observation datasets such as greenhouse gasses, air quality, and sea
-            level rise, among others.
-          </li>
-        </ul>
-        <p>
-          As an interactive visual interface for storytelling, the dashboard
-          provides a number of capabilities including sharing and interacting
-          with datasets and their respective stories. Through visualizations,
-          trend analyses, and comparative analyses of datasets users can explore
-          the applications and implications of those data. VEDA Dashboard will
-          improve the usability of the Earth science data and enable faster
-          science – one of the goals of the NASA open source science initiative.
-          Please submit any feedback or questions using the Feedback button in
-          the top right of the dashboard.
-        </p>
-      </FoldProse>
+      {aboutContent ? (
```

Comment:
Docs is recommending to use  `<ComponentOverride>` to use over-ridden component but it looks like about page is not using `ComponentOverride`? Any reason?

Reply:
So `<ComponentOverride>` would be used to override specific components in the site (like a footer). It does not show a loading placeholder (as it would be weird and tricky to have loading indicator for all occasions).

The about page is being loaded with the `MdxContent` to behave exactly like any other mdx page.

Also, at the moment, `<ComponentOverride>` does not provide any blocks to the `MdxProvider` like `MdxContent`does. Not sure we need `Blocks` when overriding components.
Would like to hear your thoughts on this.

Reply:
Ah too bad that we can't consolidate the syntax. (well but it will be mainly us who will touch this, so I think it should be just well documented.) 
Hm I am not very sure about whether to offer `Blocks` or not. but I think it will be needed for Veda dashboard considering how people are using `usage` tab now. so maybe a good idea to offer it?

Reply:
I think that the usage page would be a different thing. It seems that `usage` would be a long form page that can be different for each dataset, so the implementation would be different (like outlined in https://github.com/NASA-IMPACT/delta-ui/issues/188#issuecomment-1168576285)

Reply:
Ah right. Apology that I was confused with about page and usage page. Yeah I think we might need it, or not. We probably can add it when we need it in the future.